lua/nordic/platforms/foot.lua

```diff
@@ -0,0 +1,38 @@
+local utils = require('nordic.utils')
```

Comment:
I no longer remember the reason but we had a discussion about it and decided to continue using one character import names for modules:

```suggestion
local U = require('nordic.utils')
```

If we do decide to change it we would change it throughout the whole code-base at once.